Image processing/Image compression analyser/visualizer/diff_generator.py

```
@@ -0,0 +1,263 @@
+        """
+        width = 400
+        height = 300
+        
+        # Create visualization image
+        vis_image = Image.new('RGB', (width, height), (255, 255, 255))
+        draw = ImageDraw.Draw(vis_image)
+        
+        try:
+            font = ImageFont.truetype("arial.ttf", 14)
+            title_font = ImageFont.truetype("arial.ttf", 16)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)